app/src/main/res/values-es/strings.xml

```diff
@@ -1200,7 +1199,6 @@
     <string name="configLabel_alarms_notifications_on">Activadas</string>
     <string name="configLabel_alarms_notifications_off">Apagadas. Las alarmas pueden no mostrarse correctamente.</string>
     <string name="configLabel_alarms_notifications">Notificaciones</string>
-    <string name="configLabel_alarms_notifications_summary1">En la pantalla de bloqueo: %1%s</string>
```

Comment:
hrm, well I'm noticing that I messed this commit up - it is only half the changes. The other half are included in the 'remove duplicates' commit.